src/wms/src/test/java/org/geoserver/wms/wms_1_1_1/WMSCascadeTest.java

```diff
@@ -189,13 +186,6 @@ public void testCascadeGetLegendRequestJSON() throws Exception {
 
         print(dom);
         JSONObject responseJson = JSONObject.fromObject(dom.toString());
-        assertTrue(responseJson.has(JSONLegendGraphicBuilder.LEGEND));
```

Comment:
Can you explain what's going on here?

Reply:
yes, this was returning the local default raster style for a cascaded WMS layer configured with no specific remote style, this behavior was causing LayerGroup`s GetMap request to fail because the request internally associates the local default raster with the layer internally, which fails validation(because raster style is unknown to WMSLayerInfo resource). This occurs at 

https://github.com/geoserver/geoserver/blob/master/src/wms/src/main/java/org/geoserver/wms/GetMap.java#L691

If this is not acceptable than if incase of not finding a requested remote style, we can convert the exception into a warning and send empty string to remote request.